pkg/arguments/arguments.go

```diff
@@ -1,13 +1,43 @@
 // This file contains functions that add common arguments to the command line
-
 package arguments
 
 import (
+	"strings"
+
+	"github.com/redhat-developer/app-services-cli/internal/config"
 	"github.com/redhat-developer/app-services-cli/pkg/cmd/debug"
+	"github.com/redhat-developer/app-services-cli/pkg/cmd/factory"
 	"github.com/spf13/pflag"
 )
 
 // AddDebugFlag adds the '--debug' flag to the given set of command line flags
 func AddDebugFlag(fs *pflag.FlagSet) {
 	debug.AddFlag(fs)
 }
+
+// Enables dev preview in config
+func EnableDevPreview(f *factory.Factory, enablement *string) (*config.Config, error) {
+	if *enablement == "" {
+		// Flag not present no action needed.
+		return nil, nil
+	}
+
+	logger, err := f.Logger()
+	if err != nil {
+		logger.Info("Cannot enable dev preview")
```

Comment:
I will do i18n as last step on the feature branch. Ignore non externalized strings for now